documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -80,7 +80,7 @@ m¦monitoredPartitionsPercentage
 ¦The percentage of partitions in the Kafka cluster covered by the optimization proposal. Affected by the number of `excludedTopics`.
 
 m¦excludedTopics
-¦Not yet supported. An empty list is returned.
+¦If you have specified a regular expression in `KafkaRebalance.spec.excludedTopicsRegex` then those topics matching that regular expression will be listed here.
```

Comment:
_I think we should avoid the lengthy resource strings. Suggested rewording:_
 
excludedTopics | If you specified a regular expression in the `spec.excludedTopicsRegex` property in the `KafkaRebalance` resource, all topic names matching the regex are listed. These topics are excluded from the calculation of partition replica movements in the optimization proposal.